.github/workflows/athena_test.yml

```
@@ -96,7 +130,7 @@
         uses: actions/upload-artifact@v4
         with:
           name: junit-results
-          path: athena/test-results/*_mock.xml
+          path: athena/test-results/*_mock.xml,athena/test-results/*_real.xml
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)